docs/design/features/multilevel-sharedfx-lookup.md

```diff
@@ -208,10 +208,11 @@ Both files carry the filenames for dependencies that must be found. They can be
 
 At last, the coreclr is loaded into memory and called to run the application.
 
-### Hostpolicy changes for 2.1+
-For 2.0, there are several probing paths that are used to find the dependencies. These paths follow a certain order and the first assembly found wins and that location will be passed to the coreclr. For example, the local app location has priority over the shared framework locations and if the same assembly exists in both locations, the coreclr will end up using the local app's copy of that assembly.
+### Probing for Assemblies
+Assemblies are found by looking through probing paths in a certain order. The first found matching assembly wins, and that location will be passed to CoreCLR. For example:
 
-These semantics will be unchanged for 2.1 except when a roll-forward is performed at a non-patch version (meaning a change to the major or minor version). For these cases, the highest assembly version wins. This is necessary in run-time scenarios to prevent assembly load exceptions which occur when an assembly is referencing a higher version of another assembly, but a lower version is actually found. This situation of having assembly conflicts (or duplicates) is more likely to occur when there are multiple frameworks (as convered in hostfxr's changes for 2.1), so it is important for 2.1+ functionality.
+* In version 2.0, the local app location has priority over the shared framework locations and if the same assembly exists in both locations, the coreclr will end up using the local app's copy of that assembly.
+* In version 2.1and later, if multiple assemblies with the same name are found, the assembly with the highest version wins. This is necessary to avoid downgrading assembly versions requested by the application.
```

Comment:
```suggestion
* In version 2.1 and later, if multiple assemblies with the same name are found, the assembly with the highest version wins. This is necessary to avoid downgrading assembly versions requested by the application.
```

Reply:
@jkotas Fixed it, thanks.